main.cpp

```diff
@@ -109,7 +109,19 @@ Thing 4) Gas Grill
     3) turn of the flame
  */
 
-
+/*
+Thing 10) Wireless Headphone
+5 properties:
+    1) Speaker
+    2) Over Ear
```

Comment:
This isn't a very interesting object.  If I were you, i would choose something like `Synthesizer`, which has Oscillators, Filters, Amps, Envelopes, Effects, Modulations, etc, which all have their own set of properties and functions.
- [ ] revise

Reply:
You're right. I changed it to a drum maschine ;)